### ISSUE TITLE
[Twenty Twenty Three] inconsistent/incomplete XML import for Site Editor templates & global styles

### ISSUE THREAD

**Issue by metabreakr** [Automattic/wp-calypso#81849]
### Quick summary

With the Twenty Twenty Three (TT3) active, Site Editor templates (e.g. home, pages, single) and global styles (e.g. typography, colors, CSS) either do not import or import inconsistently between Simple and Atomic sites. 

TL;DR: Importing XML into Atomic will give you the global styles but not templates. Importing the same XML into a simple site will give you the templates but not global styles. 

More details:

* After building a site for pdEkSe-2zA-p2, I wanted to reuse customizations from the site editor in a new Atomic site. 
* When the XML was imported into an Atomic site, content was migrated. However, no templates we imported in the site editor. Global styles however did persist from the source site. Active theme on this site was TT3 1.1 by default.
* Updating to TT3 1.2 and re-importing did not yield any different results.
* Using the same XML file, I imported into a new simple site with TT3 active by default. Templates were imported, however global styles were not (basically, the opposite of the Atomic site). Content was migrated.
* After making some changes in the simple site, for testing purposes, pushing to Atomic preserved both the templates and global styles on Atomic. 

### Steps to reproduce

In both cases here, I am using an XML file exported (same day) via Calypso from a simple site that has been customized in the Site Editor. It's worth noting that XML exported from an Atomic site yields the same results.

Atomic:

1. Create a new simple site. The Twenty Twenty Three (TT3) should be active by default
2. Push the site to Atomic
3. Import the XML file via Calypso
4. Visit the Site Editor
5. Observe that while global styles (e.g. typography, colors, additional CSS) have imported, customized templates (e.g. home, page, single) have not

Simple:

1. Create a new simple site. The Twenty Twenty Three (TT3) should be active by default
2. Import the XML file via Calypso
3. Visit the Site Editor
4. Observe that while site templates have imported, global styles have not

In short, there is no way to guarantee that both Site Editor template and global styles will be imported as expected.

### What you expected to happen

With the XML import, I expected that the following Site Editor customizations  would be imported:

* templates
* global styles

### What actually happened

Depending if the site is simple or Atomic, either of the following are imported (but not both):

* templates
* global styles

### Impact

Most (> 50%)

### Available workarounds?

Yes, difficult to implement

### Platform (Simple and/or Atomic)

Simple, Atomic

### Logs or notes

One more thing that I tested:

Simple → Atomic

1. Create a new simple site. The Twenty Twenty Three (TT3) should be active by default
2. Import the XML; the Site Editor templates should have imported
4. Make some customizations to global styles in the Site Editor
5. Push the site to Atomic
6. Observe that both templates and global styles have persisted

**Comment by cuemarie** [comment]
📌 REPRODUCTION RESULTS
- Tested on Simple – Replicated 
- Tested on Atomic – Replicated
- Replicable outside of Dotcom – No

📌 FINDINGS/SCREENSHOTS/VIDEO

TLDR:
- Simple site (Premium Plan) imported template customizations, but not Global styles
- Atomic site (Business Plan) imported global styles, but not template customizations
- Self-Hosted site imported both

---
<details>
<summary><strong>Details of the export file I set up, in case it's helpful</strong></summary>
On a new simple site with TT3 installed, I added the following customizations in the site editor:

1. Global Styles > Headings blocks had Typography changes (Letter Case, Spacing)
2. Global Styles > Colors > Headings blocks set to Pink
3. Blog Home template customized with an additional heading and group block color
4. Header template part customized with an added Heading in the middle of the row

![Markup on 2023-09-26 at 12:25:51](https://github.com/Automattic/wp-calypso/assets/27249804/a53fd062-d9d3-4faf-ba4e-98bbac344a78)

I exported all via Tools > Export on this site, and used that XML file for testing.
</details>

---

**Simple** (Premium Plan Site)

Import on Simple carried over Template/Template Part customizations, but not Global Styles (pink & all caps headings):

https://github.com/Automattic/wp-calypso/assets/27249804/7c4c89f5-0344-4569-9db3-5d92944bf058

**Atomic**

Import on Atomic carried over Global Styles, but not the Template/Template Part customizations.


https://github.com/Automattic/wp-calypso/assets/27249804/183ca45e-5406-4312-9536-ea6691e15ab0

**Self-Hosted**

Both the Global Styles and the Template/Template Part changes were imported as expected.  They weren't automatically assigned to the site, but I could find the customized Blog Home template and the imported Header template part in the editor, and the Global styles carried over as well.

📌 ACTIONS
- Triage complete
- FYI: @Automattic/caribou

**Comment by mikeicode** [comment]
Closing this in favor of https://github.com/Automattic/wp-calypso/issues/86122
